### ISSUE TITLE
Private messages do not open query

### ISSUE THREAD

**Issue by anthonyryan1** [MCMrARM/revolution-irc#185]
When a user I do not have an open query with in Revolution messages me, the private message text repeatedly alerts me in a loop.

These alerts will not go away until I create a query with the user, at which point it stops. If I click on the alert, it redirects me to the server tab, rather than opening a Query and directing me there.

This is Revolution 0.5.0 connected to a znc 1.7.2. If I can do anything to help debug this issue, please don't hesitate to let me know.

**Comment by MCMrARM** [comment]
Android version and device?

**Comment by MCMrARM** [comment]
Also by a loop, how often do you mean?

**Comment by anthonyryan1** [comment]
Android 9, Pixel 2.

By looping, it happened every 3-4 seconds until I opened a query with the user. The amount of time was the time the notification displayed, before android faded it off my screen.

----

Looking at the logs on my phone, it seems I may have made a mistake in the version number. Revolution was updated yesterday at roughly the same time I opened this issue, it's likely the issue was occurring on 0.4.1 but when I looked at the version in the Play Store it updated to 0.5.0 and showed me that version number.

And now this issue is not reproducing for me. I'm going to close this issue until I have reason to suspect it's still broken. Sorry for the noise.
